File: C11/백준/Bronze/2292. 벌집/벌집.c
#include <stdio.h>

int main(void){
	
	int N;
	int term = 1;
	int cycle = 1;
	
	scanf("%d", &N);
	while(term < N){
		term += 6 * cycle;
		cycle++;
	}
	
	printf("%d", cycle);
	
	return 0;
}

File: Java/백준/Bronze/2292. 벌집/벌집.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        Scanner s = new Scanner(System.in);

        int N = s.nextInt(); //방 번호
        int term = 1, cycle = 1;

        while (term < N) {
            term += 6 * cycle;
            cycle++;
        }

        System.out.print(cycle);

        s.close();
    }
}